yt_dlp/extractor/cellebrite.py

```diff
@@ -0,0 +1,70 @@
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class CellebriteIE(InfoExtractor):
+    _VALID_URL = r'https?://cellebrite\.com/(?:\w+)?/(?P<id>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://cellebrite.com/en/collect-data-from-android-devices-with-cellebrite-ufed/',
+        'info_dict': {
+            'id': '16025876',
+            'ext': 'mp4',
+            'description': 'md5:174571cb97083fd1d457d75c684f4e2b',
+            'thumbnail': 'https://cellebrite.com/wp-content/uploads/2021/05/Chat-Capture-1024x559.png',
+            'title': 'Ask the Expert: Chat Capture - Collect Data from Android Devices in Cellebrite UFED',
+            'duration': 455,
+            'tags': [],
+        }
+    }, {
+        'url': 'https://cellebrite.com/en/how-to-lawfully-collect-the-maximum-amount-of-data-from-android-devices/',
+        'info_dict': {
+            'id': '29018255',
+            'ext': 'mp4',
+            'duration': 134,
+            'tags': [],
+            'description': 'md5:e9a3d124c7287b0b07bad2547061cacf',
+            'thumbnail': 'https://cellebrite.com/wp-content/uploads/2022/07/How-to-Lawfully-Collect-the-Maximum-Amount-of-Data-From-Android-Devices.png',
+            'title': 'Android Extractions Explained',
+        }
+    }]
+
+    def _get_formats_and_subtitles(self, json_data, display_id):
+        formats, subtitles = [], {}
+
+        mp4_url = traverse_obj(json_data, ('mp4', ..., 'url'))
+        hls_url = traverse_obj(json_data, ('hls', ..., 'url'))
+
+        for url_ in mp4_url or []:
+            formats.append({'url': url_, 'http_headers': {'Referer': 'https://play.vidyard.com/'}})
+
+        for url in hls_url or []:
+            fmt, sub = self._extract_m3u8_formats_and_subtitles(
+                url, display_id, ext='mp4', headers={'Referer': 'https://play.vidyard.com/'})
+            formats.extend(fmt)
+            self._merge_subtitles(sub, target=subtitles)
+
+        return formats, subtitles
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+
+        player_uuid = self._search_regex(
+            r'<img\s*(?:style\s*="[^"]+")?\s*(?:class\s*="[^"]+")?\s*(?:src\s*=\s*"[^"]+")?\s*data-uuid\s*=\s*"(?P<player_uuid>[^"\?]+)',
+            webpage, 'player_uuid', group='player_uuid')
```

Comment:
too strict. Is this not enough?
```suggestion
        player_uuid = self._search_regex(
            r'<img\s[^>]*\bdata-uuid\s*=\s*"([^"\?]+)', webpage, 'player UUID')
```